src/components/Modal/index.module.css

```
@@ -0,0 +1,96 @@
+  justify-content: center;
+  margin-bottom: 10px;
+  position: relative;
+}
+
+.title {
+  margin: 0;
+  font-family: var(--font-pixel);
+  font-size: 15px;
+  font-weight: 700;
+  color: var(--color-white);
+  text-shadow: 0.6px 0 currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)